eth/tracers/tracer.go

```diff
@@ -93,6 +93,15 @@ type memoryWrapper struct {
 
 // slice returns the requested range of memory as a byte slice.
 func (mw *memoryWrapper) slice(begin, end int64) []byte {
+	if end == begin {
+		return nil
```

Comment:
Nitpick, `return []byte{}` so that empty is valid and different than nil, which is errorneous.